test/remote/account_signin_verification_tests.js

```diff
@@ -447,6 +447,68 @@ TestServer.start(config)
     )
 
     test(
+      'certificate sign with keys=false session',
+      function (t) {
+        var email = server.uniqueEmail()
+        var password = 'allyourbasearebelongtous'
+        var client = null
+
+        return Client.createAndVerify(config.publicUrl, email, password, server.mailbox, {keys:true})
+          .then(
+            function (c) {
+              client = c
+              return client.emailStatus()
+            }
+          )
+          .then(
+            function (status) {
+              t.equal(status.verified, true, 'account is verified')
+              t.equal(status.emailVerified, true, 'account email is verified')
+              t.equal(status.sessionVerified, true, 'account session is  verified')
+            }
+          )
+          .then(
+            function () {
+              // Attempt to login from new location, but don't request keys
+              return client.login({keys:false})
+            }
+          )
+          .then(
+            function (c) {
+              client = c
+              return client.emailStatus()
+            }
+          )
+          .then(
+            function (status) {
+              // Ensure unverified session
+              t.equal(status.verified, false, 'account is not verified')
```

Comment:
Should this be `true`? If not requesting keys, verified would equal status of [emailVerified](https://github.com/mozilla/fxa-auth-server/blob/360cacbcae041cfa2a80d5ae67a3917f697fe603/lib/routes/account.js#L1165), which is true.